cmd/oras/internal/option/remote.go

```diff
@@ -47,6 +47,7 @@ type Remote struct {
 	Username          string
 	PasswordFromStdin bool
 	Password          string
+	IdentityToken     string
```

Comment:
Don't need this new value, the `--registry-token` can be applied to `Password` directly. The tricky part is whether we need  `--registry-token-stdin`? Please wait for https://github.com/oras-project/oras/issues/742#issuecomment-1660256192 to be answered?

Reply:
I see. I will remove the field and would implement identity-token-stdin instead.